### ISSUE TITLE
Hello

### ISSUE THREAD

**Issue by bassemhelal18** [Kodi-vStream/venom-xbmc-addons#4022]
Sorry for asking but am trying using cmultiup but I only get one link back not all of them and I don't know where the problem so if you can help.thank you in advance

**Comment by TmpName** [comment]
He ? what is cmultiup  ?

**Comment by bassemhelal18** [comment]
You can find it in lib/multihost.py
Cmultiup

**Comment by TmpName** [comment]
Huuu, it's realy old, I m not sure this version is still working.
There is a way to test this lib using Vstream ? (a site that still use it ?)

**Comment by bassemhelal18** [comment]
It's still working by added https instead of http
But the problem is it only return one link not all of them

**Comment by TmpName** [comment]
Yeah but for the moment I don't think this hoster is still used by Vstream.
If 1 link is retreived, it's a good sign, probably mean just the regex is not working, line 27 in the file for multiUp

**Comment by bassemhelal18** [comment]
No I think the regex is fine because when I use r[:-1] another link appear but also one link

**Comment by TmpName** [comment]
So if `VSlog(r)` print all values (VSlog need to be imported too) the problem is here

```
            if 'bounce-to-right' in str(item[2]) and not 'download-fast' in item[1]:
                self.list.append(item[1])
```

Can try with removing the check.

**Comment by bassemhelal18** [comment]
2023-03-18 19:16:35.953 T:6824     info <general>: 	[PLUGIN] vs: ['https://1fichier.com/?g6y7uncahwk64amdwfza&amp;af=62851', 'https://tubesb.com/3xyhco32uxnb', 'https://gofile.io/d/Lj85bV', 'https://filerio.in/y4phn5emfuuw', 'https://rapidgator.net/file/c51a89d571a6e8c6da679f5aee8c56f7/Cocaine.Bear.2023.1080p.WEB-DL.MovizLand.com.mp4.html', 'https://ddownload.com/e3vbe63sf48y', 'https://clicknupload.cc/sf6as2zg2yfi', 'https://uptobox.com/fle1nrgxxc09', 'https://mixdrop.co/f/4n0qj69liq4lmlj', 'https://doodrive.com/f/mny9p7']
2023-03-18 19:16:35.953 T:6824     info <general>: Skipped 1 duplicate messages..
2023-03-18 19:16:35.953 T:6824     info <general>: 	[PLUGIN] vStream: https://1fichier.com/?g6y7uncahwk64amdwfza&amp;af=62851
2023-03-18 19:16:35.953 T:6824     info <general>: 	[PLUGIN] vStream: https://tubesb.com/3xyhco32uxnb
2023-03-18 19:16:35.953 T:6824     info <general>: 	[PLUGIN] vStream: https://gofile.io/d/Lj85bV
2023-03-18 19:16:35.953 T:6824     info <general>: 	[PLUGIN] vStream: https://filerio.in/y4phn5emfuuw
2023-03-18 19:16:35.953 T:6824     info <general>: 	[PLUGIN] vStream: https://rapidgator.net/file/c51a89d571a6e8c6da679f5aee8c56f7/Cocaine.Bear.2023.1080p.WEB-DL.MovizLand.com.mp4.html
2023-03-18 19:16:35.953 T:6824     info <general>: 	[PLUGIN] vStream: https://ddownload.com/e3vbe63sf48y
2023-03-18 19:16:35.953 T:6824     info <general>: 	[PLUGIN] vStream: https://clicknupload.cc/sf6as2zg2yfi
2023-03-18 19:16:35.953 T:6824     info <general>: 	[PLUGIN] vStream: https://uptobox.com/fle1nrgxxc09
2023-03-18 19:16:35.953 T:6824     info <general>: 	[PLUGIN] vStream: https://mixdrop.co/f/4n0qj69liq4lmlj
2023-03-18 19:16:35.953 T:6824     info <general>: 	[PLUGIN] vStream: https://doodrive.com/f/mny9p7
2023-03-18 19:16:35.962 T:6824     info <general>: 	[PLUGIN] vStream: <resources.hosters.resolver.cHoster object at 0x000001F5BB7BE7F0>
2023-03-18 19:16:35.975 T:1776     info <general>: 	[PLUGIN] vStream: SQLITE 3 as DB engine for tmdb
2023-03-18 19:16:36.013 T:6824     info <general>: 	[PLUGIN] vStream: <resources.hosters.resolver.cHoster object at 0x000001F5BB7BEB80>
2023-03-18 19:16:37.859 T:13980    info <general>: CActiveAESink::OpenSink - initialize sink

**Comment by bassemhelal18** [comment]
hello again i have tried vslog it show that it get all the link but it check the last one only

**Comment by TmpName** [comment]
But the fonction just retrun a list, it don't check link.
I don't see where you have put your debug in the code.
If it was after 

```
            if 'bounce-to-right' in str(item[2]) and not 'download-fast' in item[1]:
                self.list.append(item[1])
```

All is fine for me

If it was after better to display all the variable r to see why the check is not working.
